src/modules/compliance/src/lib/Evaluator.cpp

```diff
@@ -1,18 +1,22 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include <string>
-#include <memory>
-#include <map>
-#include <sstream>
-#include <parson.h>
-#include <Logging.h>
 #include "Evaluator.h"
+
+#include "Logging.h"
 #include "Result.h"
 
-#define lenof(str) (sizeof(str)-1)
+#include <cstring>
+#include <map>
+#include <parson.h>
+#include <sstream>
+#include <string>
+#include <utility>
+
+#define lenof(str) (sizeof(str) - 1)
 
```

Comment:
"lenof" is not correct camelCasing or PascalCasing. We need full words for those both. 

Also, in C++, we want to have constants and inline functions as opposed to macros (which are C style).

Possible to switch this to a static global method and rename it as: 'LenghtOf'? For example